src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jobs/oneoff/concludereferrals/ConcludeReferralsJobListener.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jobs.oneoff.concludereferrals
+
+import mu.KLogging
+import net.logstash.logback.argument.StructuredArguments.kv
+import org.springframework.batch.core.BatchStatus
+import org.springframework.batch.core.JobExecution
+import org.springframework.batch.core.JobExecutionListener
+import org.springframework.stereotype.Component
+
+@Component
+class ConcludeReferralsJobListener : JobExecutionListener {
+  companion object : KLogging()
+
+  override fun beforeJob(jobExecution: JobExecution) {
+  }
+
+  override fun afterJob(jobExecution: JobExecution) {
+
+    when (jobExecution.status) {
+      BatchStatus.COMPLETED -> {
+      }
+
+      BatchStatus.FAILED -> {
+      }
+
+      else -> {
+        logger.warn(
+          "Unexpected status encountered for performance report {} {}",
```

Comment:
This doesn't seem to be related to this job. Do we need this listener at all? Would an exception be caught by Sentry by default?